src/clr-angular/accordion/accordion-panel.html

```diff
@@ -0,0 +1,48 @@
+<ng-container *ngIf="panel | async; let panel">
+  <div *ngIf="panel.status !== AccordionStatus.Inactive" aria-live="assertive" class="clr-screen-reader-only">
+    <ng-container *ngIf="panel.status === AccordionStatus.Error">{{commonStrings.danger}}</ng-container>
+    <ng-container *ngIf="panel.status === AccordionStatus.Complete">{{commonStrings.success}}</ng-container>
+  </div>
+
+  <div role="group" [ngClass]="getPanelStateClasses(panel)">
+    <div class="clr-accordion-header" [class.focus-within]="focusHeader">
+      <button
+        type="button"
+        class="clr-accordion-header-button"
+        (click)="togglePanel()"
+        (focus)="focusHeader = true"
+        (blur)="focusHeader = false"
+        [id]="'clr-accordion-header-' + panel.id + panel.accordionId"
+        [attr.aria-controls]="'clr-accordion-content-' + panel.id + panel.accordionId"
+        [attr.aria-expanded]="panel.open"
+        [disabled]="panel.disabled"
+      >
+        <div class="clr-accordion-status">
+          <clr-icon shape="angle" dir="right" class="clr-accordion-angle"></clr-icon>
+          <div class="clr-accordion-number"></div>
+          <clr-icon shape="exclamation-circle" class="clr-accordion-error-icon"></clr-icon>
+          <clr-icon shape="check-circle" class="clr-accordion-complete-icon"></clr-icon>
+        </div>
+        <ng-content select="clr-accordion-title, clr-step-title"></ng-content>
+      </button>
+      <ng-content select="clr-accordion-description, clr-step-description"></ng-content>
+    </div>
+    <div
+      @skipInitialRender
+      role="region"
+      [id]="'clr-accordion-content-' + panel.id + panel.accordionId"
```

Comment:
nit: we could benefit from having them in variables as they are being used in different areas.

Reply:
updated